src/examples/typescript/dapp/main.ts

```diff
@@ -272,6 +303,30 @@ async function hedera_signQueryAndSend(e: Event) {
 
 document.getElementById('hedera_signQueryAndSend').onsubmit = hedera_signQueryAndSend
 
+async function hedera_signQueryAndSendBatch(e: Event) {
+  const state = saveState(e)
+
+  const query1 = new AccountInfoQuery().setAccountId(state['query-payment-account']);
+  const query2 = new AccountInfoQuery().setAccountId(state['query-payment-account']);
+
+  const response: string = await signClient.request({
+    topic: activeSession.topic,
+    chainId: HederaChainId.Testnet,
+    request: {
+      method: HederaJsonRpcMethod.SignQueryAndSend,
+      params: [[
```

Comment:
Using the params like this will not scale and does not work when multiple accounts are paired